main.py

```diff
@@ -1,10 +1,47 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+
+        loaded_players = json.load(players)
+
+    for player_name, player_info in loaded_players.items():
+
+        guild_info = player_info.get("guild")
+        race_info = player_info.get("race")
+        skills = race_info.get("skills")
+
+        if guild_info:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_info.get("name"),
+                description=guild_info.get("description")
+            )
```

Comment:
You must search guild only by name, you can pass description as `defaults`. Read more: https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.query.QuerySet.aget_or_create